base/shell/explorer/startmnucust.cpp

```diff
@@ -66,16 +66,80 @@ VOID OnClearRecentItems()
         }
         while (FindNextFileW(hPath, &info));
         FindClose(hPath);
-        /* FIXME: Disable the button*/
+        
+        EnableWindow(GetDlgItem(hwnd, IDC_CLASSICSTART_CLEAR), FALSE);
     }
 }
 
+INT_PTR CALLBACK CustomizeModernProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
+{
+    switch(Message)
+    {
+        case WM_INITDIALOG:
+
+        return TRUE;
+        case WM_COMMAND:
+            switch(LOWORD(wParam))
+            {
+                case IDOK:
+                    EndDialog(hwnd, IDOK);
+                break;
+                case IDCANCEL:
+                    EndDialog(hwnd, IDCANCEL);
+                break;
+            }
+        break;
+        default:
+            return FALSE;
+    }
+    return TRUE;
+}
+
+HWND InitClassicListView(HWND hwnd)
+{
+     LVCOLUMN lvc={0};
+     RECT rcClient;
+
+     HWND hListView=GetDlgItem(hwnd, IDC_CLASSICSTART_SETTINGS);
+    ListView_SetExtendedListViewStyle(hListView, LVS_EX_CHECKBOXES| LVS_EX_FULLROWSELECT);
+
+    GetClientRect(hListView, &rcClient);
+    lvc.mask = LVCF_TEXT | LVCF_WIDTH;
+    lvc.pszText = (LPWSTR)_T("");
+    lvc.cx = rcClient.right;
+    SendMessage(hListView, LVM_INSERTCOLUMN, 0, (LPARAM)&lvc);
+
+    LVITEM hitem={0};
+    hitem.mask=LVIF_TEXT;
+    TCHAR chBuffer[64];
+    int id=IDS_DRAG_DROP;
+
+    for(int i=0;i<12;i++)
+    {
+        hitem.iItem=i;
+        LoadStringW(NULL, id++, chBuffer, 60);
+        hitem.pszText=chBuffer;
+        hitem.cchTextMax=60;
+        SendMessage(hListView,LVM_INSERTITEM,0,(LPARAM)&hitem);
+    }
+
+    ListView_SetCheckState(hListView,0, TRUE);
+    ListView_SetCheckState(hListView,10, TRUE);
```

Comment:
Hardcoded values.

Reply:
OK Maybe this selection is on the register!